builtin/tests/commit_post_test.rs

```diff
@@ -0,0 +1,649 @@
+    //                 ExpectInvocation {
+    //                     to: REWARD_ACTOR_ADDR,
+    //                     method: RewardMethod::ThisEpochReward as u64,
+    //                     ..Default::default()
+    //                 },
+    //                 FIXME: This invocation is missing?
+    //                 ExpectInvocation {
+    //                     to: miner_id,
+    //                     method: MinerMethod::ConfirmSectorProofsValid as u64,
+    //                     subinvocs: Some(vec![ExpectInvocation {
+    //                         to: STORAGE_POWER_ACTOR_ADDR,
+    //                         method: PowerMethod::UpdatePledgeTotal as u64,
+    //                         ..Default::default()
+    //                     }]),
+    //                     ..Default::default()
+    //                 },
+    //                 ExpectInvocation {
+    //                     to: REWARD_ACTOR_ADDR,
+    //                     method: RewardMethod::UpdateNetworkKPI as u64,
+    //                     ..Default::default()
+    //                 },
+    //             ]),
+    //             ..Default::default()
+    //         },
+    //         ExpectInvocation {
+    //             to: STORAGE_MARKET_ACTOR_ADDR,
+    //             method: MarketMethod::CronTick as u64,
+    //             ..Default::default()
+    //         },
+    //     ]),
+    //     ..Default::default()
+    // }
+    // .matches(v.take_invocations().last().unwrap());
+
+    // pcd is released ip is added
+    let balances = get_miner_balance(&mut w, miner_id);
+    assert!(balances.initial_pledge.is_positive());
+    assert!(balances.pre_commit_deposit.is_zero());
+
+    println!("PCD verified, initial pledge is positive {}", balances.initial_pledge);
+
+    // power unproven so network stats are the same
+
+    // let network_stats = v.get_network_stats();
+    // assert!(network_stats.total_bytes_committed.is_zero());
+    // assert!(network_stats.total_pledge_collateral.is_positive());
+
+    let (deadline_info, partition_index) =
+        advance_to_proving_deadline(&mut w, &miner_addr, sector_number);
+
+    println!("Setup complete");
+
+    (
+        w,
+        MinerInfo {
+            seal_proof,
+            worker: worker.id_addr(),
+            _owner: owner.id_addr(),
+            miner_id: Address::new_id(miner_id),
+            _miner_robust: miner_addr,
+        },
+        SectorInfo { number: sector_number, deadline_info, partition_index },
+    )
+}
+
+fn get_miner_balance(w: &mut ExecutionWrangler, miner_id: ActorID) -> MinerBalances {
+    let a = w.find_actor(miner_id).unwrap().unwrap();
+    let st: MinerState = w.find_actor_state(miner_id).unwrap().unwrap();
+    MinerBalances {
+        available_balance: st.get_available_balance(&a.balance).unwrap(),
+        vesting_balance: st.locked_funds,
+        initial_pledge: st.initial_pledge,
+        pre_commit_deposit: st.pre_commit_deposits,
+    }
+}
```

Comment:
Maybe we need a new `util/inspections` or similar package for various state inspection things like this.